Deploy-CAM.ps1

```diff
@@ -2812,6 +2909,56 @@ else {
         }
     } while ( -not $domainAdminCredential )
 
+    $radiusConfig = @{
+        enableRadiusMfa = $enableRadiusMfa
+        radiusServerHost = $radiusServerHost
+        radiusServerPort = $radiusServerPort 
+        radiusSharedSecret = $radiusSharedSecret
+    }
+    if ( ($enableRadiusMfa -eq $null) -or $enableRadiusMfa ) {
+        if ( $enableRadiusMfa -eq $null -and (-not $ignorePrompts) ) {
+            $enableRadiusMfa = (Read-Host "Do you want to enable Multi-Factor Authentication using your Radius Server? (yes/no)") -like "*y*"
+        } elseif ( $enableRadiusMfa -eq $null -and $ignorePrompts ) {
+            $enableRadiusMfa = $false
+        }
+
+        if ($enableRadiusMfa) {
+            if (-not $radiusConfig.radiusServerHost ) {
+                $radiusConfig.radiusServerHost = Read-Host "Please enter your Radius Server's Hostname or IP"
+            }
+
+            do {
+                if (-not $radiusConfig.radiusServerPort ) {
+                    try {
+                        $radiusConfig.radiusServerPort = [int](Read-Host  "Please enter your Radius Server's Listening port")
+                    } catch {
+                        $radiusConfig.radiusServerPort = $null
+                        Write-Host "Selected port is not an Integer"
+                    }
+                    if ( ($radiusConfig.radiusServerPort -le 0) -or ($radiusConfig.radiusServerPort -gt 65535) ) {
```

Comment:
If you pop this 'if' up one level it will also check passed parameters, not just command-line inputs (see the other do loops in the code.)